bokehjs/src/coffee/node_main.coffee

```diff
@@ -0,0 +1,43 @@
+path = require "path"
+assert = require "assert"
+rootRequire = require("root-require")
+
+root = rootRequire.packpath.parent()
+pkg = rootRequire("./package.json")
+
+module.constructor.prototype.require = (modulePath) ->
+  assert(modulePath, 'missing path')
+  assert(typeof modulePath == 'string', 'path must be a string')
+
+  load = (modulePath) =>
+    this.constructor._load(modulePath, this)
+
+  overridePath = pkg.browser[modulePath]
+
+  if overridePath?
+    modulePath = path.join(root, overridePath)
+
+  return load(modulePath)
+
+if not (global.window? and global.document?)
+  jsdom = require('jsdom').jsdom
```

Comment:
According to webpack's documentation, it should be possible to avoid this custom `require()` logic by setting `resolve.packageAlias = "browser"`.